src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -201,6 +201,15 @@ export const Checkout: React.FC<Props> = ({
 
   const scrollWrapper = useRef<HTMLDivElement>()
 
+  useEffect(() => {
+    if (visibilityState === VisibilityState.OPEN) {
+      Intercom('update', { hide_default_launcher: true })
+    }
+    if (visibilityState === VisibilityState.CLOSED) {
+      Intercom('update', { hide_default_launcher: false })
+    }
+  }, [visibilityState])
+
```

Comment:
Awesome! 🙌

Reply:
😊 And this could easily be made into a custom hook if we want to hide the Intercom launcher somewhere else in the future.